props/saber_fett263_buttons.h

```diff
@@ -257,7 +331,96 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+#ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+         if (!battle_mode_) {
+           battle_mode_ = true;
+           hybrid_font.SB_Force();
+         } else {
+           battle_mode_ = false;
+           hybrid_font.SB_Force();
+         }
+         return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+         if (!battle_mode_) return false;
+         clash_impact_millis_ = millis();
+         swing_blast_ = false;
+         if (!swinging_) {
+           SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+           swing_blast_ = false;
+           auto_lockup_on_ = true;
+           SaberBase::DoBeginLockup();
+           return true;
+	 }
+
+       case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON):
+         if (!battle_mode_) return false;
+         clash_impact_millis_ = millis();
+         swing_blast_ = false;
+         if (!swinging_) {
+           if (fusor.angle1() < - M_PI / 4) {
+             SaberBase::SetLockup(SaberBase::LOCKUP_DRAG);
+             } else {
+               SaberBase::SetLockup(SaberBase::LOCKUP_MELT);
+       	     }
+         swing_blast_ = false;
+         auto_melt_on_ = true;
+         SaberBase::DoBeginLockup();
+         return true;
+         }
 	
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_OFF):  
+         if(millis() > 3000) { 
```

Comment:
Please add a comment to explain this, it looks weird, but I think maybe it's correct?
Although shouldn't swing-on be controlled by a define?

Reply:
I added the define back in

Reply:
it's the millis() > 3000 that looks kind of odd.
I assume that it's to prevent this happening on boot?  I suggest we add a comment to explain that's what it's for.
Also, you need a space after the "if".